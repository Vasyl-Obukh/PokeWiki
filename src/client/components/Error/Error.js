import styled from 'styled-components';

const Error = styled.h2`
  padding-left: 25px;
  
  &::after {
    content: '!!!';
    padding-left: 5px;
    color: red;
  }
`;

export default Error;