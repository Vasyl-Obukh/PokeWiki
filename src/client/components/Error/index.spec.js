import React from 'react';
import Enzyme, { mount } from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';
import renderer from 'react-test-renderer';
import Error from './index';

Enzyme.configure({ adapter: new Adapter() });

describe('<Error /> test', () => {
  const errorMessage = 'Something goes wrong...';

  test('text passing test', () => {
    const component = mount(<Error>{errorMessage}</Error>);
    expect(component.text()).toBe(errorMessage);
  });

  test('snapshot test', () => {
    const component = renderer.create(<Error>{errorMessage}</Error>);
    const tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });
});