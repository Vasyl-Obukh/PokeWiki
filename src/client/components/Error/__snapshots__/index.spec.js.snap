// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Error /> test snapshot test 1`] = `
<h2
  className="sc-bdVaJa gnWSe"
>
  Something goes wrong...
</h2>
`;
